packages/sdk/src/packageMetadata.ts

```diff
@@ -2,4 +2,4 @@
 // Licensed under the MIT license.
 
 export const name = "@microsoft/teamsfx";
-export const version = "0.1.3";
+export const version = "0.2.0-alpha.0";
```

Comment:
not blocking this PR: can we book sometime next sprint to improve the dev experience around this file?

Reply:
Yes, it should also be automatically updated in a daily CI job.